Doc/whatsnew/2.4.rst

```diff
@@ -995,7 +995,7 @@ fixes.  Here's a partial list of the most notable changes, sorted alphabetically
 by module name. Consult the :file:`Misc/NEWS` file in the source tree for a more
 complete list of changes, or look through the CVS logs for all the details.
 
-* The :mod:`asyncore` module's :func:`loop` function now has a *count* parameter
+* The :mod:`!asyncore` module's :func:`loop` function now has a *count* parameter
```

Comment:
The removed module doesn't have a loop function anymore, because it doesn't have anything anymore 😈

```suggestion
* The :mod:`!asyncore` module's :func:`!loop` function now has a *count* parameter
```